### PR TITLE
Race rank logic

### PR DESCRIPTION
**Pull request by ryanbaumann**
@Rayhan96Shuvo created this PR to make some comments on your code to address questions about how to validate the logic in the `calculateRiderRank` along route function.

### CODE REVIEW

File: src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx [hunk 1]
```diff
@@ -1,28 +1,46 @@
 import * as _ from 'lodash';
 import * as React from "react";
 
+import WebMercatorViewport from 'viewport-mercator-project';
+import bbox from '@turf/bbox';
+import * as turf from '@turf/turf'
```
Comment (ryanbaumann) [inline src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx:6]: Turf.js as a whole is a big library, I recommend importing these two specific modules to make your page faster -

`import helpers, nearest-point-on-line as turf from '@turf/turf'`

File: src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
+import imgDavePlusPin from 'assets/images/personPinPlus.png';
+
 
 import styles from './RaceTrackerMap.module.css';
+// import { number } from 'prop-types';
+
 
 
 
 const TOKEN = "pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqdzg5aWxkYzF1azI0OW5uaWVmazhleXUifQ.XAm1dRGmXuRAMSQm0TJKXg";
```
Comment (ryanbaumann) [inline src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx:39]: Update to `pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqd281d2M3bzF4MHczeXA5dm9xdXg4bjAifQ._vwACgp12mC4RBpuSYl1CA`

**Review by ryanbaumann — COMMENTED** [review]
@Rayhan96Shuvo small comments - logic looks reasonable at a glance.  If you can describe exactly what you want to achieve and where the logic problem area is I'll take a closer look.

File: src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx (hunk 2 above)
Comment (andrew-ironforge) [reply src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx:39]: @ryanbaumann @Rayhan96Shuvo I will do this and push to master. Do not do this in this branch.

**Comment by andrew-ironforge** [comment]
Merge master

File: src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx (hunk 2 above)
Comment (ryanbaumann) [reply src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx:39]: no sweat.  Just opened this PR to give feedback on a few q's that @Rayhan96Shuvo had in Slack

File: src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx (hunk 1 above)
Comment (Rayhan96Shuvo) [reply src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx:6]: Added some comments around the func rankRiders.

**Pull request merged**